README.md

```diff
@@ -262,6 +262,22 @@ magic methods will be striginfied in JSON as simple objects) and you will not be
 features like mixed indent, pre/post render hooks but in this mode you will get exact same
 output as in pug-js.
 
+### Write locals object to json file with pugjs
+
+If your locals object is large it may cause a `RuntimeException` error. This is because
+locals object passed directly to pug-cli as argument. To fix this problem you can use
+`localsJsonFile` option:
+
+```php
+$pug = new Pug(array(
+    'expressionLanguage' => 'js',
```

Comment:
expressionLanguage is automatically js and cannot be changed if pugjs => true, so this option should be omitted here.